src/commands/diff.ts

```diff
@@ -125,6 +128,10 @@ export default class Diff extends Command {
         err: error,
       });
     }
+    if (watchMode) {
```

Comment:
I think we also need conditions for `firstDocument.getFilePath()` and `secondDocument.getFilePath()` right? because if they are passed as url, then watched doesn't make much sense.

This also makes me think that we most probably should probably log a message to the use that in case AsyncAPI file path is passed as URL than watching doesn't start and we say `sorry boss, not this time` 😄

Reply:
Last thing, rest is good :)

We should move that `if` at the begging of logic like in `validation` command, after 67 line :)